modules/placement-driver/src/test/java/org/apache/ignite/internal/placementdriver/leases/LeaseBatchSerializerTest.java

```diff
@@ -94,10 +100,33 @@ void batchWithTablePartitionsOnly() {
                         false,
                         false,
                         null,
-                        new TablePartitionId(2, 1)
+                        groupIdFactory.apply(2, 1)
                 )
         );
+    }
+
+    private static Stream<Arguments> leasesSource() {
+        return Stream.of(
+            arguments(createLeases(baseTs(), TablePartitionId::new)),
+            arguments(createLeases(baseTs(), ZonePartitionId::new)),
+            arguments(
+                    Stream.concat(
+                            createLeases(baseTs(), TablePartitionId::new).stream(),
+                            createLeases(baseTs(), ZonePartitionId::new).stream()
+                    ).collect(toList())
+            ),
+            arguments(
+                    Stream.concat(
+                            createLeases(baseTs(), ZonePartitionId::new).stream(),
+                            createLeases(baseTs(), TablePartitionId::new).stream()
+                    ).collect(toList())
+            )
+        );
+    }
 
```

Comment:
I would suggest to add an analog of v1CanBeDeserialized() for a batch containing zone partition leases